doc/spire_server.md

```diff
@@ -358,6 +362,72 @@ Deletes bundle data for a trust domain. This command cannot be used to delete th
 | `-mode`       | One of: `restrict`, `dissociate`, `delete`. `restrict` prevents the bundle from being deleted if it is associated to registration entries (i.e. federated with). `dissociate` allows the bundle to be deleted and removes the association from registration entries. `delete` deletes the bundle as well as associated registration entries. | `restrict` |
 | `-socketPath` | Path to the SPIRE Server API socket | /tmp/spire-server/private/api.sock |
 
+### `spire-server federation create`
+
+Creates a dynamic federation relationship with a foreign trust domain.
+
+| Command       | Action                                                             | Default        |
+|:--------------|:-------------------------------------------------------------------|:---------------|
+| `-bundleEndpointProfile` | Endpoint profile type. Either `https_web` or `https_spiffe`. | |
+| `-bundleEndpointURL` | URL of the SPIFFE bundle endpoint that provides the trust bundle (must use the HTTPS protocol). | |
+| `-data` | Path to a file containing federation relationships in JSON format (optional). If set to '-', read the JSON from stdin. | |
```

Comment:
Should we somehow mention that `data` is mutually exclusive with the rest of the flags?